src/can-deploy.ts

```diff
@@ -124,11 +110,14 @@ export class CanDeploy {
 
 export default (options: CanDeployOptions) => new CanDeploy(options);
 
-export interface CanDeployOptions extends SpawnArguments {
-  participant?: string;
-  participantVersion?: string;
-  to?: string;
```

Comment:
Is there a reason why we removed this?

Reply:
These were moved into the `canDeployPacticipant` type.

I believe `--to=TAG` and `--latest=TAG` are synonyms, and are supported in the new interface by specifying a tag. 

I considered supporting the old API as well, but figured that it would be a lot of effort (and complexity) for little pay off (as you pointed out).

Reply:
@bethesque is that correct about synonyms?

Reply:
Nope. --to=TAG is for the deployment environment. --latest=TAG is for specifying the version of the application that you're about to deploy.

eg 
```
can-i-deploy --pacticipant Foo --latest --to prod (can I deploy the latest version of Foo to prod)
can-i-deploy --pacticipant Foo --latest test --to prod (can I deploy the latest version of Foo with tag test to prod)
```

Reply:
https://github.com/pact-foundation/pact_broker-client#can-i-deploy

I would imagine most people would use the single pacticipant form of this command, but I have no way to know actual numbers.

Reply:
Hi,
Chiming in with my use-case regarding this change.
I initially attempted to use `canDeploy` but was disappointed that it only supported a single pacticipant so this would be a welcome change for me. I switched over to calling the cli `can-i-deploy` within a child process in my program in order to accomplish it currently. I am using the `latest` option in order to determine if some version of Foo is compatible with the latest marked prod version of Bar:  
`can-i-deploy --pacticipant Foo --version 123 --pacticipant Bar --latest prod`
but I also use the `to` option similar to the example given by @bethesque so I believe this option still needs to be distinctly supported

Reply:
Thanks @kuangp

Reply:
I'd say "determine compatibility with all the services in a deployment environment" rather than "configure compatibility".

Reply:
I can't find the documentation to update it (I can't see the source in the standalone repo, I can only see it on the releases page), but if they're not synonyms I think we should change the lines:

```
  -l, [--latest=[TAG]] 
      [--to=TAG] 
````

to

```
  -l, [--latest=[TAG]] 
  --to=TAG 
```

Reply:
The `--to TAG` is not required if you've specified the other pacticipants explicitly.